monsterWebNew.py

```
@@ -0,0 +1,331 @@
+
+
+def motor_stop_callback() -> None:
+    """Emergency stop motor callback."""
+    global TB
+    if TB is not None:
+        TB.MotorsOff()
+        TB.SetLedShowBattery(False)
+        TB.SetLeds(1, 0, 0)  # Red LED
+
+
+def on_emergency_state_change(is_stopped: bool, reason: str) -> None:
```

Ruff (0.14.8)

Unused function argument: `reason`

(ARG001)